zone/bot.h

```diff
@@ -789,6 +793,7 @@ class Bot : public NPC {
 	bool _pauseAI;
 	uint8 _stopMeleeLevel;
 	int m_expansion_bitmask;
+	bool _enforcespellsettings;
```

Comment:
I know the old ones don't follow the same standard, but we try to prefix member variables with `m_` like `m_enforce_spell_settings`.